### PR TITLE
Add urllib3 to Discord OSS list

### PR DESCRIPTION
**Pull request by sethmlarson**
Let me know if the SVG is acceptable.

### CODE REVIEW

File: communities.json [hunk 1]
```diff
@@ -771,6 +771,13 @@
       "quote": "Discord has been amazing for accelerating the development of React Spring and React Three Fiber. Screen sharing and instant communication have been pivotal in resolving bugs and coordinating with our contributors.",
       "inviteCode": "poimandres",
       "githubUrl": "https://github.com/pmndrs"
+    },
+    {
+      "logo": "urllib3.svg",
+      "title": "urllib3",
+      "quote": "The urllib3 team uses Discord to answer questions from users and to collaborate on development tasks.",
+      "inviteCode": "urllib3",
```
Comment (jamiebuilds) [inline communities.json:779]: It looks like this invite code doesn't currently exist. I cannot identify your server without a valid invite code.

Comment (sethmlarson) [reply communities.json:779]: Oh sorry! I thought this was an aspirational field :)

Comment (sethmlarson) [reply communities.json]: Updated!

**Comment by jamiebuilds** [comment]
Turned on some features for you:

- [Vanity Invite URL](https://support.discord.com/hc/en-us/articles/115001542132-Server-Vanity-URLs)
- [Server Banner & Invite Splash](https://support.discord.com/hc/en-us/articles/360028716472-Server-Banner-Background-Invite-Splash-Image)

Do you want to update your invite code to something like `discord.gg/<yourprojectname>`?

File: communities.json (hunk 1 above)
Comment (jamiebuilds) [reply communities.json]: Haha sorry, I should have just said to comment it. I've turned on vanity urls so you could set it back now

**Comment by sethmlarson** [comment]
@jamiebuilds Thanks much! I've switched the invite code to `urllib3`

**Review by jamiebuilds — APPROVED** [review]

**Pull request closed without merging**